lib/shelley/test/data/cardano-node-shelley/regenerate-byron.sh

```diff
@@ -29,21 +29,20 @@ cat > byron.genesis.spec.json <<EOF
 }
 EOF
 
-cardano-cli genesis \
+cardano-cli byron genesis genesis \
    --protocol-magic 764824073 \
    --start-time 2500000 \
    --k 8 \
    --n-poor-addresses 0 \
    --n-delegate-addresses 1 \
    --total-balance 50 \
-   --byron-formats \
    --delegate-share 1 \
    --avvm-entry-count 0 \
    --avvm-entry-balance 0 \
    --protocol-parameters-file byron.genesis.spec.json \
    --genesis-output-dir tmp
 
-yq -y . < tmp/genesis.json > byron-genesis-init.yaml
+cat tmp/genesis.json | yq e -P - > byron-genesis-init.yaml
```

Comment:
This is a very different `yq` from the one I have... maybe let's just use json.

```suggestion
jq . tmp/genesis.json > byron-genesis-init.yaml
```

Reply:
Oh, it seems there are two:
 - https://kislyuk.github.io/yq/
 - https://mikefarah.gitbook.io/yq/

I had the latter, going back to the first one... (as it seems it is used here and there across different scripts)